.github/pull_request_template.md

```diff
@@ -1,7 +1,13 @@
-# Description
-Please include a summary of the change. For large or complex changes please include enough information to introduce your change and explain motivation for it.
+### Description 
+_Add a comprehensive description of proposed changes_
 
-Changes proposed in this pull request:
--
--
--
\ No newline at end of file
+Fixes # - _issue number(s) if exists_
```

Comment:
i would suggest you to split this to several different templates - MKL as an example - https://github.com/oneapi-src/oneMKL/issues/new/choose

Reply:
Addressed on #2903